apps/fluent-tester/src/E2E/FocusZone/pages/FocusZonePageObject.ts

```diff
@@ -1,12 +1,112 @@
-import { FOCUSZONE_TESTPAGE, FOCUSZONE_TEST_COMPONENT, HOMEPAGE_FOCUSZONE_BUTTON } from '../../../TestComponents/FocusZone/consts';
+import { FocusZoneDirection } from '@fluentui/react-native';
+import { Attribute, AttributeValue, Keys } from '../../common/consts';
+import {
+  FOCUSZONE_CIRCLE_NAV_SWITCH,
+  FOCUSZONE_DEFAULT_TABBABLE_SWITCH,
+  FOCUSZONE_DIRECTION_ID,
+  FOCUSZONE_DISABLED_SWITCH,
+  FOCUSZONE_GRID_AFTER,
+  FOCUSZONE_GRID_BEFORE,
+  FOCUSZONE_GRID_BUTTON,
+  FOCUSZONE_TESTPAGE,
+  FOCUSZONE_TEST_COMPONENT,
+  FOCUSZONE_TWO_DIM_SWITCH,
+  HOMEPAGE_FOCUSZONE_BUTTON,
+} from '../../../TestComponents/FocusZone/consts';
 import { BasePage, By } from '../../common/BasePage';
 
+export const enum GridFocusZoneOption {
+  SetDirection = 0,
+  Set2DNavigation,
+  SetCircularNavigation,
+  UseDefaultTabbableElement,
+  Disable,
+}
+
+export const enum GridButtonSelector {
+  Before = 0,
+  One,
+  Two,
+  Three,
+  Four,
+  Five,
+  Six,
+  Seven,
+  Eight,
+  Nine,
+  After,
+}
+
+type BooleanGridFocusZoneOption =
+  | GridFocusZoneOption.UseDefaultTabbableElement
+  | GridFocusZoneOption.Set2DNavigation
+  | GridFocusZoneOption.SetCircularNavigation
+  | GridFocusZoneOption.Disable;
+
 class FocusZonePageObject extends BasePage {
-  // OVERRIDE: We use isExisting() here instead of isDisplayed() because FocusZone does not have any UI to it, it's simply
-  // a wrapper that adds keyboard focus functionality
   async waitForPrimaryElementDisplayed(timeout?: number): Promise<void> {
     const errorMsg = 'The FocusZone UI Element did not load correctly. Please see logs.';
-    await this.waitForCondition(async () => await this._primaryComponent.isExisting(), errorMsg, timeout);
+    await this.waitForCondition(async () => await this._primaryComponent.isDisplayed(), errorMsg, timeout);
```

Comment:
Should this be changed back to isExisting()? (per the comment you omitted above?)

Reply:
This is targeting a Text Element above the focus zone, since FocusZone doesn't allow for a testID to be attached. My logic is that if the text element is visible, then the FocusZone should also be visible below, though I'm not even sure if this is necessary.